Mytinerary/View Controllers/DailyCalendarViewController.m

```diff
@@ -226,11 +226,13 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     if ([[segue identifier] isEqualToString:@"eventDetailsSegue"]) {
         EventDetailsViewController *eventDetailsViewController = [segue destinationViewController];
         eventDetailsViewController.event = sender;
```

Comment:
Hey just wondering  where we use this segue. Maybe we do not need this line to assign the eventDetailsViewController.event to a sender.